src/components/refactor/dialogs/shunt-compensator-creation/shunt-compensator-creation-dialog.js

```diff
@@ -217,7 +218,8 @@ ShuntCompensatorCreationDialog.propTypes = {
         then: PropTypes.func.isRequired,
         catch: PropTypes.func.isRequired,
     }),
-    currentNodeUuid: PropTypes.string,
+    studyUuid: PropTypes.string,
+    currentNode: PropTypes.string,
```

Comment:
change type to object instead of string
```suggestion
    currentNode: PropTypes.object,
```